demo/demo.md

```diff
@@ -68,6 +68,46 @@ This example illustrates a mainline nonstop flight from SEA to EWR (SEAEWR)
 
 </auro-accordion>
 
+## Header Slot Example
+
+This example illustrates a mainline nonstop flight from SEA to EWR (SEAEWR)
```

Comment:
The example is showing more than a flight from SEA to EWR. There is no information to point the reader to the fact that the additional slots are being used and why.